### ISSUE TITLE
Sails creating new Nunjucks instance after update to 0.11.0 

### ISSUE THREAD

**Issue by etx** [balderdashy/sails#2680]
I'm defining a ton of extensions and filters in a config/nunjucks.js file by requiring nunjucks and getting a new env, which I thought was a singleton. But it was working fine in 0.10.5. 

After updating to 0.11.0 I can see two instances are being created and the one sails is using does not have my custom extensions. Looks like this second instance of nunjucks is not created until the first template is rendered. 

How can I get a reference to this instance, or force Sails to use my instance I create with my custom config on startup?

Thanks for the great framework!

**Comment by CWyrtzen** [comment]
@etx do you have any updates to this?

**Comment by sailsbot** [comment]
Thanks for posting, @etx.  I'm a repo bot-- nice to meet you!

It has been 60 days since there have been any updates or new comments on this page.  If this issue has been resolved, feel free to disregard the rest of this message.  On the other hand, if you are still waiting on a patch, please:

  + review our [contribution guide](https://github.com/balderdashy/sails/blob/master/CONTRIBUTING.md) to make sure this submission meets our criteria (only _verified bugs_ with documented features, please;  no questions, commentary, or bug reports about undocumented features or unofficial plugins)
  + create a new issue with the latest information, including updated version details with error messages, failing tests, and a link back to [the original issue]().  This allows GitHub to automatically create a back-reference for future visitors arriving from search engines.

Thanks so much for your help!
